### PR TITLE
add --turbconv_case cli

### PR DESCRIPTION
**Pull request by yairchn**
I am adding here a `turbconv_case_name` so that we can added several other cases

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ Insolation = "e98cc03f-d57e-4e3c-b70c-8d51efe9e0d8"
 IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NNlib = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
```
Comment (charleskawczynski) [inline Project.toml:16]: ?

File: post_processing/mse_tables.jl [hunk 2]
```diff
@@ -57,17 +57,17 @@ all_best_mse["sphere_held_suarez_rhoe_int_equilmoist"][(:c, :uₕ, :components,
...
-all_best_mse["edmf_single_column"][(:c, :ρe_tot)] = 0.05010398752695285
-all_best_mse["edmf_single_column"][(:c, :uₕ, :components, :data, 1)] = 15.633092708261552
-all_best_mse["edmf_single_column"][(:c, :uₕ, :components, :data, 2)] = 81.25287663937327
-all_best_mse["edmf_single_column"][(:c, :ρq_tot)] = 0.3567000179447617
-all_best_mse["edmf_single_column"][(:c, :turbconv, :en, :ρatke)] = 99.56872201272839
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρarea)] = 7.884506143708434
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρaθ_liq_ice)] = 7.888499297457295
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρaq_tot)] = 7.866565945507953
-all_best_mse["edmf_single_column"][(:f, :turbconv, :up, 1, :ρaw)] = 2.8234965025701935
+all_best_mse["TurbulenceConvection"] = OrderedCollections.OrderedDict()
```
Comment (charleskawczynski) [inline post_processing/mse_tables.jl:60]: The keys correspond to the `job_id` specified in the buildkite yml

Comment (charleskawczynski) [reply post_processing/mse_tables.jl:60]: So, we'll need to add new dicts for the new jobs (if we opt in to regression tests).

**Comment by yairchn** [comment]
Bors r+

**Review by charleskawczynski — COMMENTED** [review]
It looks like we forgot to increment the ref counter

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**